server/@types/licenceApiImport/index.d.ts

```diff
@@ -4,6 +4,10 @@
  */
 
 export interface paths {
+  '/licence/id/{licenceId}/appointmentTime': {
+    /** Update the date and time for the initial appointmen. Requires ROLE_SYSTEM_USER or ROLE_CVL_ADMIN. */
```

Comment:
I'll fix this typo in the API

Reply:
Already done it in my next PR for contact number! Coming shortly.